data/transactions/logic/doc.go

```diff
@@ -377,8 +377,8 @@ func OpDocExtra(opName string) string {
 var OpGroups = map[string][]string{
 	"Arithmetic":              {"sha256", "keccak256", "sha512_256", "sha3_256", "ed25519verify", "ed25519verify_bare", "ecdsa_verify", "ecdsa_pk_recover", "ecdsa_pk_decompress", "vrf_verify", "bn256_add", "bn256_scalar_mul", "bn256_pairing", "+", "-", "/", "*", "<", ">", "<=", ">=", "&&", "||", "shl", "shr", "sqrt", "bitlen", "exp", "==", "!=", "!", "len", "itob", "btoi", "%", "|", "&", "^", "~", "mulw", "addw", "divw", "divmodw", "expw", "getbit", "setbit", "getbyte", "setbyte", "concat"},
 	"Byte Array Manipulation": {"substring", "substring3", "extract", "extract3", "extract_uint16", "extract_uint32", "extract_uint64", "replace2", "replace3", "base64_decode", "json_ref"},
-	"Byte Array Arithmetic":   {"b+", "b-", "b/", "b*", "b<", "b>", "b<=", "b>=", "b==", "b!=", "b%", "bsqrt"},
-	"Byte Array Logic":        {"b|", "b&", "b^", "b~"},
+	"Byte Array Arithmetic":   {"b+", "b-", "b/", "b*", "b|", "b&", "b^", "b~", "b%", "bsqrt"},
+	"Byte Array Logic":        {"b<", "b>", "b<=", "b>=", "b==", "b!="},
```

Comment:
moved `b` ops that return an int to `Logic` and `b` ops that produce bytes to `Arithmetic`